src/app/public/modules/autonumeric/option-guards.ts

```diff
@@ -0,0 +1,6 @@
+import { SkyAutonumericOptions } from './autonumeric-options';
+import AutoNumeric from 'autonumeric';
+
+export function isPredefinedAutoNumericOption(option: SkyAutonumericOptions): option is string | keyof AutoNumeric.PredefinedOptions {
+  return typeof option === 'string';
```

Comment:
I was porting what was originally in the directive but I agree that this can be better. 
- Updated the type guard 
- Added tests for confidence.

![image](https://user-images.githubusercontent.com/10877317/118440060-1ace5e00-b6b5-11eb-86a9-6b005289e6f7.png)